src/clojure_lsp/feature/hover.clj

```diff
@@ -34,6 +34,7 @@
 (defn hover-documentation [{sym-ns :ns sym-name :name :keys [doc filename arglist-strs] :as _definition}]
   (let [[content-format] (get-in @db/db [:client-capabilities :text-document :hover :content-format])
         show-docs-arity-on-same-line? (get-in @db/db [:settings :show-docs-arity-on-same-line?])
+        hide-filename? (get-in @db/db [:settings :hide-filename?])
```

Comment:
could you change to:
```suggestion
        hide-filename? (get-in @db/db [:settings :hover :hide-file-location?])
```

Reply:
How should that be reflected in `docs/settings.md`?

Reply:
I think this is enough: https://github.com/clojure-lsp/clojure-lsp/pull/499/files#r680416292